specs/darwin/connected_displays.table

```diff
@@ -0,0 +1,21 @@
+table_name("connected_displays")
+description("Provides information about the connected displays of the machine.")
+schema([
+    Column("name", TEXT, "The name of the display."),
+    Column("product_id", TEXT, "The product ID of the display."),
+    Column("serial_number", TEXT, "The serial number of the display."), 
+    Column("vendor_id", TEXT, "The vendor ID of the display."),
+    Column("display_week", INTEGER, "The manufacture week of the display. This field is 0 if not supported"), 
+    Column("display_year", INTEGER, "The manufacture year of the display. This field is 0 if not supported"),
```

Comment:
These names sound weird to me. Maybe `manufactured_week` and `manufactured_year`? Or maybe combine them to `manufactured_date`?